### ISSUE TITLE
Variables not showing

### ISSUE THREAD

**Issue by Proladon** [Almenon/AREPL-vscode#330]
**Python : 3.6.8**

Variables not showing

like in picture example, variable `a` not showing on AREPL panel

![image](https://user-images.githubusercontent.com/35193493/81060565-4f85d200-8f05-11ea-8312-b961a588b24b.png)

**Comment by Almenon** [comment]
That's very odd. The variables show for me with your code running with python 3.6.4, and it should work for python 3.6.8 too.

If you go to help, toggle developer tools, and reproduce the problem does any information appear in the console?

**Comment by Proladon** [comment]
> any information appear in the console?

aw...too late.

everything works fine today. yes, it's really odd!
guess maybe stuck with cache or something else idk...
if i'v got same problem again, i'll post to here ASAP.

**Comment by Proladon** [comment]
I got same issue so quick. Lol

![02](https://user-images.githubusercontent.com/35193493/81180032-0f921e00-8fdd-11ea-9757-96db08199b10.gif)

after that it show:
![image](https://user-images.githubusercontent.com/35193493/81180017-0a34d380-8fdd-11ea-960b-d413e6a13861.png)

**Comment by Almenon** [comment]
Thanks for the image! Renderjson is the library I use to render the variables as JSON, so if it's not defined that explains why not variables are showing up. 

It's really weird that it would be undefined. There's not much code logic that could go wrong that would cause that. I just instantiate the variable 
```typescript
this.jsonRendererScript = `<script src="${this.getMediaPath("jsonRenderer.js")}"></script>`
```
 and put it into the html. It's pretty simple logic. I wonder if `getMediaPath` failed to find the script somehow.

I can also see in the console logs that arepl is executing, so it looks like the backend is still working 🤔 

Can you go to help -> about, click copy, and paste the information here please? It will tell me what vscode version you're using and your OS version.

> it's only show the other Extention: Power Mode Error
so maybe is Extension conflict?

I don't think so. I was able to run arepl and power mode together, the power mode error appeared but arepl still worked.

> btw. i can't use the developer tool to inspect AREPL tab element, so i don't know how to css Print Output font-size, only i knew it is #257 span { font-size: 40px; } but it only change Variables font-size

To inspect the arepl tab element you have to use the `Developer: Open Webview Developer Tools` command. Control-shift-p to activate the command prompt. The print results have a CSS class `print` so you can target that with `.print` like so: `.print { font-size: 40px; }`

Speaking of which, next time the error happens can you run the webview developer tools command and take a look around please? If you can send me the html or a image of the html it might clarify what is going on. 

If you use the selector on the top left 

https://i.imgur.com/8dl3f2z.png

to select the `10` print output and then expand the head element the html should look like this:

https://i.imgur.com/VIQRXBN.png

Images of the console / network tab would also be helpful. Note that the network tab will be blank when you first go to it - you have to trigger another arepl run and then it should record the network traffic.

**Comment by Proladon** [comment]
```
Version: 1.44.2 (user setup)
Commit: ff915844119ce9485abfe8aa9076ec76b5300ddd
Date: 2020-04-16T16:36:23.138Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18362
```

> next time the error happens...

Sure !

> you have to use the Developer: Open Webview Developer Tools

Oh! 
I was used `Help > Toggle Developer Tools`, and it can't inspect AREPL
idk there have two different ways Developer Tools. Thanks for help~❤

**Comment by Proladon** [comment]
I think i found the way that how to trigger the issue...
very simple, just `Help > Toggle Developer Tools` and Boom💥
(`Ctrl+Shift+P :Open Webview Developer Tools` not effect)
here's record:
<https://youtu.be/IVHuiXzURow>

![image](https://user-images.githubusercontent.com/35193493/81266006-9a732700-9076-11ea-964a-ce50ccaefec5.png)

I didn't remembered the first time is it trigger the same way or not.... sorry
but overall this is way to 100% trigger the issue on my vscode

**Comment by Almenon** [comment]
Thanks for the video! That's very useful.

Unfortunately I have bad news - I'm still unable to reproduce the problem. Futhermore, I googled the error and it appears to be a internal vscode error :(

https://github.com/microsoft/vscode/issues/89038

Hopefully it will be fixed soon, but it's been in the backlog for months so might be a while...

The only good news I have is that the unicode error that happened to you in the video was already reported in https://github.com/Almenon/AREPL-vscode/issues/149 and it will be fixed next release.

For now as a temporary solution you can create a .env with `PYTHONIOENCODING=utf8`

Sorry I'll not able to help out more.

**Comment by Proladon** [comment]
Nope, Thanks for help and keep going.
AREPL is really awsome!

**Comment by Almenon** [comment]
https://github.com/microsoft/vscode/pull/97777 should fix this. It's in insiders at the moment, hopefully testing goes well and it gets released next month :)

**Comment by Almenon** [comment]
Note that if you're already using vscode insiders you will also have to wait for a new version of my extension - I updated it to use a new webview scheme and have to do a release.

**Comment by Almenon** [comment]
Next release should be coming shortly. Just need a teeny tiny bit more testing

**Comment by Almenon** [comment]
I just released AREPL 1.0.24. If you're on vscode insiders hopefully this problem will no longer appear.
